src/overlays/actors/ovl_En_Baba/z_en_baba.c

```diff
@@ -5,29 +5,50 @@
  */
 
 #include "z_en_baba.h"
-#include "overlays/actors/ovl_En_Ossan/z_en_ossan.h"
-#include "objects/object_bba/object_bba.h"
 
 #define FLAGS (ACTOR_FLAG_1 | ACTOR_FLAG_8 | ACTOR_FLAG_10)
 
 #define THIS ((EnBaba*)thisx)
 
+#define ENBABA_END_CONVERSATION (1 << 0)
+#define ENBABA_VISIBLE (1 << 1)
+#define ENBABA_KNOCKED_OVER (1 << 2) // Don't track player
```

Comment:
Notably these first three do not fit the subS pattern.

Reply:
`EnBaba` doesn't make use of that system at all.